web-ui/src/main/resources/catalog/js/edit/EditorController.js

```diff
@@ -454,8 +454,16 @@
           // when the editor was not opened by a script.
         }
 
-        // Go to editor home
-        $location.path('');
+        var lastPage = document.referrer;
```

Comment:
Cant' we simply do history.back(); ? so if we have a link to the editor from the admin or a third party app, it goes back to the last page ?

Reply:
Well i wanted to redirect to md view if coming from grid result of search page.
Maybe i can redirect to history.back() in the else statement ?

Reply:
history.back will not return to md view ? it should as the uuid is part of the URL no